test cases/frameworks/1 boost/test_python_module.py

```diff
@@ -0,0 +1,22 @@
+import sys
+sys.path.append(sys.argv[1])
+
+# import compiled python module depending on version of python we are running with
+if sys.version_info[0] == 2:
+    import python2_module as pm
+
+if sys.version_info[0] == 3:
+    import python3_module as pm
```

Comment:
__[Flake8]__ [F811] redefinition of unused 'pm' from line 6 ([link](https://sider.review/gh/repos/19784232/pulls/5596#issue-4309849))

<sub>posted by Sider</sub>